Vagrantfile

```diff
@@ -27,21 +46,21 @@ if File.exists? VM_SETTINGS_FILE
 else
 
   # Project settings
-  VM_PROJECT = ENV['VM_PROJECT'] || 'suite'                                # Project name
+  VM_PROJECT = ENV['VM_PROJECT'] || 'suite-nonsplit'                                # Project name
```

Comment:
invalid change, see other pr.

Reply:
you can still init by command in docs 
https://spryker.atlassian.net/wiki/spaces/DIO/pages/2836660432/DevVM+-+v4.1.0
example
VM_PROJECT=suite SPRYKER_REPOSITORY="git@github.com:spryker-shop/b2b-demo-shop.git" vagrant up
will override it

Reply:
No, the other way around, internally, we can do this, the default MUST be customer driven

Reply:
np changing

Reply:
Please make sure it works out of the box with public setup, master suite
and can be easily configured on top for internal use (Spryker devs)